app/models/evidence.rb

```diff
@@ -34,4 +36,14 @@ def local_fields
       'Title' => issue.try(:title)
     }
   end
+
+  private
+
+  def issue_cannot_be_on_another_project
+    return unless node && issue
+
+    if node.project.id != issue.project.id
```

Comment:
we can just use `if project.id != issue.project.id` as evidence has a delegate method for project to the node.project.